### DISCUSSION TITLE
Is there a way to search in both directions?

### DISCUSSION THREAD

**Discussion by alisnic** [ggandor/leap.nvim#47] [Q&A] [answered]
I find myself having a mental stutter every time I want to engage Leap in order to understand where the target text is in relation to my cursor. Having a hotkey to search in both directions would be awesome

**Comment by ggandor** [comment, marked as answer]
There is, see the [FAQ](https://github.com/ggandor/leap.nvim#faq) 😉

**Reply by alisnic** [reply]
Omg I feel stupid 🤦
